src/samples/func_tools/json/session_01/02.json

```diff
@@ -0,0 +1,62 @@
+{
+  "responseId": "d30932ec-d909-40e2-ba52-533fd2b299d9",
+  "queryResult": {
+    "responseMessages": [
+      {
+        "responseType": "RESPONSE_TYPE_UNSPECIFIED",
```

Comment:
According to [latest documentation](https://cloud.google.com/dialogflow/vertex/docs/concept/tools) we should be getting something similar to:
```json
{
        "source": "VIRTUAL_AGENT",
        "toolCall": {
          "tool": "<tool-resource-name>",
          "action": "get-weather-tool",
          "inputParameters": {
            "location": "Mountain View"
          }
        }
      }
``` 

When trying to detect a further intent we get the following error:
```
Error: 3 INVALID_ARGUMENT: 
com.google.apps.framework.request.BadRequestException: 
Session is waiting for tool call result of tool, 05edea2d-dbc2-4ed9-ba62-c0c50dcf2450, and action, TestToolVision1.
```
which shows that the session has paused and waits for a tool result invocation.